web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -77,6 +77,40 @@ describe("<SidebarCardsContainer />", () => {
   };
 
   describe("nudges", () => {
+    it("renders a card in the top section if set to above-opportunities", () => {
+      const sidebarCards = {
+        "funding-nudge": generateSidebarCardContent({
+          id: "funding-nudge",
+          section: "above-opportunities",
+        }),
+      };
+
+      useMockBusiness({
+        preferences: generatePreferences({ visibleSidebarCards: ["funding-nudge"] }),
+      });
+
+      renderPage({ sidebarCards });
+      const topCardSection = screen.getByTestId("top-cards");
+      expect(within(topCardSection).getByTestId("funding-nudge")).toBeInTheDocument();
+    });
+
+    it("renders a card in the bottom section if set to below-opportunities", () => {
+      const sidebarCards = {
+        "formation-nudge": generateSidebarCardContent({
+          id: "formation-nudge",
+          section: "below-opportunities",
+        }),
+      };
+
+      useMockBusiness({
+        preferences: generatePreferences({ visibleSidebarCards: ["formation-nudge"] }),
+      });
+
+      renderPage({ sidebarCards });
+      const bottomCardSection = screen.getByTestId("bottom-cards");
+      expect(within(bottomCardSection).getByTestId("formation-nudge")).toBeInTheDocument();
+    });
```

Comment:
I think we're still missing the third test, which was called "renders the welcome card" but in the body, it was just testing that the content of a card rendered on the page, so I'd argue it should be added back and renamed something like "it renders content for visible card"

Reply:
oh that's a good catch. I think to your point above, we should create test content so it's not confusing because I read renders welcome card and fully thought it was only specific to that card only.

Reply:
@aloverso can you re review :)